### ISSUE TITLE
Linker error in moveit_setup_assistant using from source install instructions

### ISSUE THREAD

**Issue by skohlbr** [moveit/moveit_ros#220]
Building MoveIt! from source as described in the installation instructions on the wiki currently fails due to a linker error in moveit_setup_assistant:

<pre>
Linking CXX executable /home/kohlbrecher/moveit_tmp/devel/lib/moveit_setup_assistant/moveit_setup_assistant
/home/kohlbrecher/moveit_tmp/devel/lib/libmoveit_robot_model.so: undefined reference to `moveit::Profiler::start()'
/home/kohlbrecher/moveit_tmp/devel/lib/libmoveit_robot_model.so: undefined reference to `moveit::Profiler::stop()'
/home/kohlbrecher/moveit_tmp/devel/lib/libmoveit_robot_model.so: undefined reference to `moveit::Profiler::end(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
/home/kohlbrecher/moveit_tmp/devel/lib/libmoveit_robot_model.so: undefined reference to `moveit::Profiler::Instance()'
/home/kohlbrecher/moveit_tmp/devel/lib/libmoveit_robot_model.so: undefined reference to `moveit::Profiler::begin(std::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)'
collect2: ld returned 1 exit status
</pre>

**Comment by skohlbr** [comment]
Just realized this could be directly ticketed in the moveit_setup_assistant issue tracker, feel free to MoveIt! ;) there.

**Comment by davetcoleman** [comment]
This is not a Setup Assistant issue but a moveit_core/robot_model issue. I'll wait for @isucan for the official fix but it looks like [this commit](https://github.com/ros-planning/moveit_core/commit/768030e582bcc11155d1ec8b3824971d1aa3dbc3) broke the build with the profiler. A temp fix is to do this:

```
roscd moveit_core
git co 510aa1af0bd6229763c04234791f678f5c474056
```

To undo that bad commit.

**Comment by isucan** [comment]
Sorry about that -- I forgot to link a lib. Should be fixed now 06fb40fa07570d2b2cbf3da58c70a1e4c2c36a1c
